### DISCUSSION TITLE
Is chart editor not available in PK-Sim while conducting population simulations?

### DISCUSSION THREAD

**Discussion by humblewarrior** [Open-Systems-Pharmacology/Forum#789] [Questions & Problems] [answered]
![PK-Sim](https://user-images.githubusercontent.com/44484877/99202258-7490b280-277c-11eb-90d0-2d08585be638.jpg)
![Mobi](https://user-images.githubusercontent.com/44484877/99202261-765a7600-277c-11eb-83ed-6f9699644c93.jpg)

Hi!

I really appreciate that PK-Sim has really nice chart editor when I create an individual simulation (in top image) which allows us to modify axes, units, and pick time scale. But, I noticed that when I create population simulation (bottom pic), I don't have that same functionality of chart editor to edit my graphs right there. Am I missing something?

**Comment by msevestre** [comment]
@humblewarrior Both pictures are taken from PK-Sim. One is an individual simulation the other one is for a population simulation
Please explain what you mean

**Comment by humblewarrior** [comment]
@msevestre I will definitely keep in mind to post future discussions in the forum. I am extremely sorry, once again.

So, in an individual simulation (in top image), you see chart editor on right-hand side of the pic, which allows you modify your graph in variety of ways e.g. pick legend position, change color coding, change labels, change axis scales, pick a time scale etc etc.
However, when I do population simulation (in the bottom pic), you can see that chart options now don't have that functionality to allow me to modify my graph in a variety of ways.

**Comment by Ricardo-DO** [comment, marked as answer]
Hi

Probably the options of the population simulations are not as flexible as for the individual data but you can set the characteristics at the beginning of the simulation.

![pop1](https://user-images.githubusercontent.com/59565610/100469175-ae8f7c00-30cd-11eb-866e-bdc1fdca459c.png)

In the output tab you can change the colour of the lines, the scale, units and which statistics to plot.

I hope that helps.

Regards,
Ricardo

**Comment by GrazielaLock** [comment]
Hello, as I read in the comments above, the graphics of the population simulation are not customizable. By the way, can we remove the grid lines from the y and x-axis? I tried to do it on R, but I just found information about how to perform simulations and create the new individual like doing my PBPK model again, but I just want to export the data from PKSim in a dataset from Excel and customize the graphics removing the grid lines (doesn't make sense to me, performing new simulations and modeling something that I just modeled on the software). So, is it possible just remove the grid lines without performing the simulations again? For example, I have a graph (below) from a population simulation and want without the grid as the individual simulation. 
![population](https://github.com/Open-Systems-Pharmacology/Forum/assets/75985275/f14d0460-c1b5-4cc8-8639-700d72fb7894)
![individual](https://github.com/Open-Systems-Pharmacology/Forum/assets/75985275/24b28bb5-b235-454d-a03e-42aee8323df0)

Thank you for your help!
Graziela

**Reply by Ricardo-DO** [reply]
Hello Graziela,

I think you cannot remove the grid (maybe someone knows better than me).
But what you can do is just export the results to excel and create there your own plot. You don't need to recreate the whole simulation in R.

For example, I have plotted the median, and range from 2.5 to 97.5%.

Just right click on the simulation and Export to Excel
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/a51edbd2-e1f0-445e-8831-795587bac63e)


You will have a file like this
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/c3d76ce0-5f20-425c-ab87-d08e5b23863a)

And then you can create your customised plot
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/26b182f3-c257-46dc-bf32-0e50a52c1c56)

I don't know how to get the grey area in excel, but you can get it easily with R using polygon with the plot function. Probably there is an easier way to do it with ggplot2.

Hope it was helpful.

Cheers,
Ricardo
